scripts/run-functional-tests.ps1

```diff
@@ -11,6 +11,14 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+Param (
+  [string]$BaseImageName="microsoft/windowsservercore"
+)
+
+# Prepared base image
+Invoke-Expression "docker pull $BaseImageName"
```

Comment:
I'd do an existence check first. If the image is cached, the tests go way faster.

Reply:
Will do.